### PR TITLE
Navbar 671

### PR DESCRIPTION
**Pull request by jacquie**
This fixes some of the issues in bug 671.

### CODE REVIEW

File: projects/cashmere-examples/src/lib/cashmere-example.module.ts [hunk 1]
```diff
@@ -0,0 +1,358 @@
+
```
Comment (benjanderson) [inline projects/cashmere-examples/src/lib/cashmere-example.module.ts]: @joeskeen is this supposed to be checked in?

File: projects/cashmere/src/lib/navbar/navbar.component.spec.ts [hunk 2]
```diff
@@ -43,38 +43,43 @@ describe('NavbarComponent', () => {
     it('should create', () => {
```
Comment (benjanderson) [inline projects/cashmere/src/lib/navbar/navbar.component.spec.ts]: Very nice tests!

File: projects/cashmere-examples/src/lib/cashmere-example.module.ts (hunk 1 above)
Comment (joeskeen) [reply projects/cashmere-examples/src/lib/cashmere-example.module.ts]: No, this should not be checked in.  It should be deleted.

Comment (jacquie) [reply projects/cashmere-examples/src/lib/cashmere-example.module.ts]: Thanks - I was wondering!

Comment (jacquie) [reply projects/cashmere-examples/src/lib/cashmere-example.module.ts]: Should it be added to .gitignore?

Comment (jacquie) [reply projects/cashmere-examples/src/lib/cashmere-example.module.ts]: I removed it

File: projects/cashmere/src/lib/navbar/navbar.component.html [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
-    <div class="hc-navbar-link-container">
+    <div #navlinks class="hc-navbar-link-container">
         <ng-content select="hc-navbar-link"></ng-content>
         <div class="hc-navbar-more-links" title="More" *ngIf="_collapse" [hcPopover]="navbarMore" popperPlacement="bottom">
             More<hc-icon fontSet="fa" fontIcon="fa-chevron-down" hcIconSm></hc-icon>
         </div>
     </div>
 
-    <div class="hc-navbar-right-container">
+    <div #rightcontainer class="hc-navbar-right-container">
```
Comment (jonest) [inline projects/cashmere/src/lib/navbar/navbar.component.html]: Should we prefix any of these ids? I could see a possible collision on this one

File: projects/cashmere-examples/src/lib/cashmere-example.module.ts (hunk 1 above)
Comment (joeskeen) [reply projects/cashmere-examples/src/lib/cashmere-example.module.ts]: You shouldn't need to add it to .gitignore.  The new example generation process doesn't create this file.

Also, don't forget to push!

File: projects/cashmere/src/lib/navbar/navbar.component.html (hunk 3 above)
Comment (joeskeen) [reply projects/cashmere/src/lib/navbar/navbar.component.html]: This is an Angular variable name, and is scoped into the component.  There won't be any collisions (that would result in a compile error).

Comment (andrew-frueh) [reply projects/cashmere/src/lib/navbar/navbar.component.html]: That's a good call @jonest - maybe #hc-rightcontainer

Comment (jonest) [reply projects/cashmere/src/lib/navbar/navbar.component.html]: Doh! Good call @joeskeen :) I completely mixed the two up :) I saw the `#` and thought "jQuery ID tag!" 😆

Comment (joeskeen) [reply projects/cashmere/src/lib/navbar/navbar.component.html]: Again, not necessary - it's just a template variable.  I don't think you can even put hyphens in those.

**Pull request merged**